tlj-db/README.txt

```diff
@@ -0,0 +1,11 @@
+Create SQL database:
```

Comment:
Move it file to root of project and change it in properly(work) way. Rename to README.md